packages/web/src/shared/bibleTranslationClient.ts

```diff
@@ -26,6 +35,42 @@ class BibleTranslationClient {
       name: name_local ? name_local : name_english,
     }));
   }
+
+  async getTranslation(
+    verseId: string,
+    translationIds: string[]
+  ): Promise<BibleVerseTranslation | undefined> {
+    const { bookId, chapterNumber, verseNumber } = parseVerseId(verseId);
+    const bookKey = bookKeys[bookId - 1].toLowerCase();
+    const collection = await this.client.fetch_collection();
+    const translations = await collection.get_translations();
+    for (const translationId of translationIds) {
+      try {
+        const book = await collection.fetch_book(translationId, bookKey, 'txt');
+        const verseTranslation = book.get_verse(chapterNumber, verseNumber, {
+          attribute: false,
+          verse_nums: false,
+          headings: false,
+          notes: false,
+        });
+        const translation = translations.find((t) => t.id === translationId);
+        if (translation) {
+          const { name_local, name_english } = translation;
+          return {
+            name: name_local ? name_local : name_english,
+            translation: verseTranslation,
+          };
+        }
+      } catch (e) {
+        console.log(e);
+        // There was some issue getting the verse in this translation, try the
+        // next translation.
+        continue;
+      }
+    }
+    // Must return null, not undefined, so that this will work with useQuery
+    return undefined;
```

Comment:
Do we need to turn this back to `null` for the sake of `useQuery`?

Reply:
Oops. It looks like I somehow missed some changes in the last commit